src/vertexAttribute.js

```diff
@@ -21,7 +21,8 @@ vgl.vertexAttributeKeys = {
   'Normal' : 1,
   'TextureCoordinate' : 2,
   'Color' : 3,
-  'CountAttributeIndex' : 4
+  'Scalar': 4,
+  'CountAttributeIndex' : 5
```

Comment:
Do we really need this in the sense that would this be a common attribute that folks would like to use?